requirements.txt

```
@@ -1,3 +1,3 @@
 pygame
-ppb-vector
+ppb-vector == 0.3
```

```suggestion
ppb-vector < 1.0.0
```

This matches what I did in the 0.5.1 release.